lib/ansible/module_utils/redfish_utils.py

```diff
@@ -511,6 +511,32 @@ def update_user_password(self, user):
             return response
         return {'ret': True}
 
+    def get_supported_firmware_update_methods(self):
+        result = {}
+        response = self.get_request(self.root_uri + "/redfish/v1")
+
+        if response['ret'] is False:
+            return response
+
+        result['ret'] = True
+
+        result['entries'] = []
+        data = response['data']
+        if "UpdateService" in data:
+            response = self.get_request(self.root_uri + "/redfish/v1/UpdateService")
```

Comment:
Rather than hardcode "/redfish/v1/UpdateService", better to read the value of the "@odata.id" property from the "UpdateService" property.